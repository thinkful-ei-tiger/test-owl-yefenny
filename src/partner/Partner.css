.partner {
    display: flex;
    flex-direction: column;
    justify-content: center;
    margin-bottom: 20px;
}
.header{
    width: 150px;
    background-color: #373a40;
    text-align: center;
    color: white;
    margin: 0;
    border-radius: 3%;
    align-self: center;
    
}
.header h2{
   
    font-size: 14px;
}

.icons img {
    width: 20px;
    height: 20px;
    margin: 10px;
}

.avatar {
    align-self: center;
    border-radius: 50%;
}

.avatar img{
    border-radius: 50%;
    width: 200px;
}