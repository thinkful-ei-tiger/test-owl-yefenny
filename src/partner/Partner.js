import React, { Component } from 'react';
import volume from '../images/volume.svg';
import search from '../images/search.svg';
import './Partner.css';

export default class Partner extends Component {
  render() {
    return (
      <div className='partner'>
        <div className='header'>
          <h2>{this.props.name}</h2>
          <div className='icons'>
            <span>
              <img src={volume} alt='mute' />
              <img src={search} alt='search' />
            </span>
          </div>
        </div>
        <div className='avatar'>
          <img src={this.props.avatar} alt='profile' />
        </div>
      </div>
    );
  }
}
