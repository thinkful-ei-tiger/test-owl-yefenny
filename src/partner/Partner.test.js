import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Partner from './Partner';

describe('Partner component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Partner />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('UI renders without problem', () => {
    const tree = renderer
      .create(
        <Partner
          name='Koren Templeton'
          avatar='https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1'
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
