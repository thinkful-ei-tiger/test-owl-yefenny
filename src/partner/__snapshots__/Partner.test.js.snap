// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Partner component UI renders without problem 1`] = `
<div
  className="partner"
>
  <div
    className="header"
  >
    <h2>
      Koren Templeton
    </h2>
    <div
      className="icons"
    >
      <span>
        <img
          alt="mute"
          src="volume.svg"
        />
        <img
          alt="search"
          src="search.svg"
        />
      </span>
    </div>
  </div>
  <div
    className="avatar"
  >
    <img
      alt="profile"
      src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
    />
  </div>
</div>
`;
